react/features/lobby/components/native/styles.js

```diff
@@ -0,0 +1,132 @@
+// @flow
+
+const SECONDARY_COLOR = '#B8C7E0';
```

Comment:
Shouddln't this be elsewhere?

Reply:
If you mean color scheming, we agreed to get rid of scheming, as we use the branded colors anyhow, and dark mode will change everything, I think